wikithingsdb/query.py

```diff
@@ -0,0 +1,203 @@
+import sys
+
+from peewee import fn
+
+from wikithingsdb.create import get_hypernyms
+from wikithingsdb.models import Article, Type, WikiClass, DbpediaClass,\
+    ArticleClass, ArticleType, Hypernym
+from wikithingsdb.util import to_wikipedia_class, to_dbpedia_class, \
+    from_dbpedia_class
+
+
+def types_of_article(article, limit=sys.maxint):
```

Comment:
It's a common pattern. In math and algorithms you initialize variables as `+infinity` or `-infinity`. When you implement these in code it translates to `sys.maxint` and `-sys.maxint-1`.